docs/api/session.md

```diff
@@ -1091,6 +1091,45 @@ app.whenReady().then(() => {
 })
 ```
 
+#### `ses.setUSBProtectedClassesHandler(handler)`
+
+* `handler` Function\<string[]> | null
+  * `details` Object
+    * `protectedClasses` string[] - The current list of protected USB classes. Possible class values are:
+      * `audio`
+      * `audio_video`
+      * `hid`
+      * `mass_storage`
+      * `smart_card`
+      * `video`
+      * `wireless`
+
+Sets the handler which can be used to override which USB classes are protected.
+Returning an empty string array from the handler will allow all USB classes; returning the passed in array will maintain the default list of protected USB classes (this is also the default behavior if a handler is not defined).
+To clear the handler, call `setUSBProtectedClassesHandler(null)`.
+
+```javascript
+const { app, BrowserWindow } = require('electron')
+
+let win = null
+
+app.whenReady().then(() => {
+  win = new BrowserWindow()
+
+  win.webContents.session.setUSBProtectedClassesHandler((details) => {
+    // Allow all classes
+    // return []
+    // Keep the current set of protected classes
```

Comment:
I'm a little confused by this - is this the possible set of options for handling permissions here?

Reply:
@codebytere I'm showing different ways developers could use this handler.    The possible set of return values are an empty array or an array containing any of the following values:
      * `audio`
      * `audio_video`
      * `hid`
      * `mass_storage`
      * `smart_card`
      * `video`
      * `wireless`